syft/formats/common/spdxhelpers/to_format_model_test.go

```diff
@@ -169,93 +168,6 @@ func Test_toFileChecksums(t *testing.T) {
 	}
 }
 
-func Test_fileIDsForPackage(t *testing.T) {
```

Comment:
This has been deprecated in favor of using the `CONTAINS` relationship, which we are already including. See: http://spdx.org/rdf/terms#relationshipType_contains